src/Router.js

```diff
@@ -14,10 +15,11 @@ const Router = () => {
           <Route path="/" element={<App />} />
           <Route path="/home" element={<Home />} />
           <Route path="/Login" element={<Login />} />
+          <Route path="/section-main" element={<SectionMain />} />
         </Routes>
       </BrowserRouter>
     </div>
-  );
-};
+  )
+}
 
-export { Router };
```

Comment:
It is much better to use semicolons, it makes the code easier to read by making it obvious where the code ends.